source/includes/_userprovisioning.md

```diff
@@ -0,0 +1,142 @@
+# User Provisioning
+
+Please note that the base URL for user provisioning is
+
+`https://services.1brd.com`
+
+## Ingest User Profiles
+
+> This is an example request body
+
+```json
+{
+  "users": [ 
+    {
+      "first_name":"Test",
+      "last_name":"User",
+      "personnel_number":"1234",
+      "email_address":"test.user@firstbird.com",
+      "location":"Vienna",
+      "department":"IT"
+    },
+    {
+      "first_name":"Test",
+      "last_name":"User2",
+      "personnel_number":"1235",
+      "email_address":"test.user2@firstbird.com"
+    }
+  ]
+}
+```
+
+You can use this endpoint to send multiple `UserInformation` objects as a list of `users`.
+This will stage all users for provisioning.
+
+### HTTP Request
+
+`POST /v1/companies/{companyId}/users/ingest`
```

Comment:
Is this really the really the path of the customer service?

Reply:
yes